src/core/qfield.h.in

```diff
@@ -58,6 +58,7 @@ namespace qfield
   static inline const QString appName{ QStringLiteral( "@APP_NAME@" ) };
 }; // namespace qfield
 
+#define WITH_VCPKG @WITH_VCPKG@
```

Comment:
That looks like a workaround. What are you trying to solve?

Reply:
It's not really a workaround but more of an attempt at leveraging the flexibility of vcpkg's bsckporting of features through patches while not forcing a non-vcpkg minimum QGIS version to be QGIS dev branch. 

With vcpkg, we're in full control of what QGIS is built with. We want to stick to stable 3.32 (which means _QGIS_VERSION would be 332000. But we're pushing a backport through patch, hence the WITH_VCPKG.

That way, we get the best of all worlds.

Reply:
Can we just patch the qgis version number to 332001 instead?